### ISSUE TITLE
New instruction `string.from_code_point`

### ISSUE THREAD

**Issue by Liedtke** [WebAssembly/stringref#59]
Following up on https://github.com/WebAssembly/stringref/issues/5#issuecomment-1312572737 we decided to implement `string.from_code_point` for performance evaluations in V8 (see also #58):
- `string.from_code_point(codepoint: i32) -> ref string`, opcode `0xfba9`
- returns a string consisting of the provided codepoint
- the instruction traps if the argument is not a valid unicode codepoint

**Comment by gkdn** [comment]
Shouldn't this be returning `ref string` instead? 
@Liedtke  @jakobkummerow

**Comment by gkdn** [comment]
BTW, I think also binaryen mismatches Chrome here and assumes the instruction returns `ref string` which results in code rejected by V8. Adding @kripken to verify.

**Comment by Liedtke** [comment]
@gkdn Yeah, the returned value is never null. I'll adapt the return type in v8 and in my comment above.

**Comment by MaxGraey** [comment]
What should happen for invalid code points? Like `0x110000` and above. In JavaScript this leads to an exception. At least for `String.fromCodePoint`

**Comment by Liedtke** [comment]
@MaxGraey The instruction traps for invalid codepoints.

**Comment by gkdn** [comment]
@Liedtke do you have link to the dashboard to track the release? or if you know dev-channel version to get the fix that would work as well.

**Comment by jakobkummerow** [comment]
@gkdn It's in Dev 112.0.5612, see https://chromiumdash.appspot.com/commit/54b016989406901e819d3090a92739e7bc1c98b9.
